### PR TITLE
Make modal-body wider so scroll bars are not truncated

### PR DESCRIPTION
**Pull request by redmunds**
This is for internal issue 366 and https://github.com/adobe/brackets-edge-web-fonts/issues/158.

### CODE REVIEW

**Comment by couzteau** [comment]
Hey @redmunds, I can confirm that the scrollbars are visible now, but there are 2 other issues with the EWF dialog:
- dialog witdh
- clipping and positioning

I have updated bug #366 in the edge code repo.

**Comment by redmunds** [comment]
Fixed remaining issues.

For dialog width, I added `width: 535px;` to `.edge-web-fonts-browse-dialog` rule to override `width: 600px;` in `.modal` base rule. Also had to remove `width: 520px;` from `.modal-header` which is not necessary.

For clipping and positioning, I removed `margin-top: -275px;` from `.edge-web-fonts-browse-dialog`. Modal dialog now automatically centers dialogs vertically, so this is no longer necessary.

Ready for review.

File: styles/ewf-brackets.css [hunk 1]
```diff
@@ -28,9 +28,8 @@
     line-height: 40px;
 }
 
-.edge-web-fonts-browse-dialog .modal-body,
-.edge-web-fonts-browse-dialog .modal-header {
-  width: 505px;
+.edge-web-fonts-browse-dialog .modal-body {
+    width: 520px;
```
Comment (couzteau) [inline styles/ewf-brackets.css:32]: consolidate with rule below? Both are for .edge-web-fonts-browse-dialog .modal-body

**Comment by couzteau** [comment]
Dialog looks good now. Done with review.

File: styles/ewf-brackets.css (hunk 1 above)
Comment (redmunds) [inline styles/ewf-brackets.css:32]: Good catch! Fixed.

**Comment by redmunds** [comment]
Ready for re-review.

**Comment by couzteau** [comment]
Merging.

**Comment by couzteau** [comment]
Now  we need to update the SHA for EWF. Will set up a pull request

**Pull request merged**